### PR TITLE
Update protocol to use webicq v14

### PR DESCRIPTION
**Pull request by leameow**
A few fixes while debugging why I stopped receiving message. I moved `startSession` to similar calls than in the new web client. This also simplify code, not using query signature.

### CODE REVIEW

File: libicyque.c [hunk 1]
```diff
@@ -28,10 +28,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
-#define ICQ_API_SERVER        "https://api.icq.net"
+#define ICQ_EVENTS             "myInfo,presence,buddylist,typing,hiddenChat,hist,mchat,sentIM,imState,dataIM,offlineIM,userAddedToBuddyList,service,lifestream,apps,permitDeny,diff" // ,webrtcMsg

+#define ICQ_PRESENCE_FIELDS    "aimId,displayId,friendly,friendlyName,state,userType,statusMsg,statusTime,lastseen,ssl,mute,abContactName,abPhoneNumber,abPhones,official,quiet,autoAddition,largeIconId,nick,userState"

+#define ICQ_ASSERT_CAPS "094613584C7F11D18222444553540000,0946135C4C7F11D18222444553540000,0946135b4c7f11d18222444553540000,0946135E4C7F11D18222444553540000,AABC2A1AF270424598B36993C6231952,1f99494e76cbc880215d6aeab8e42268"

+#define ICQ_API_SERVER        "https://api.icq.net"

+#define ICQ_API14_SERVER      "https://u.icq.net/api/v14/wim"
```
Comment (EionRobb) [inline libicyque.c:35]: Are the `^M`s from the different line endings?

File: libicyque.c [hunk 2]
```diff
@@ -1618,36 +1621,24 @@ icq_session_start(IcyQueAccount *ia)
...
+	g_string_append_printf(url, "k=%s&", purple_url_encode(ICQ_DEVID));
+	g_string_append(url, "view=online&");
+	g_string_append(url, "clientName=webicq&");
+	g_string_append(url, "language=en-US&");
+	g_string_append_printf(url, "deviceId=%s&", purple_url_encode(ia->device_id));
+	g_string_append(url, "sessionTimeout=31536000&");
+	g_string_append_printf(url, "assertCaps=%s&", purple_url_encode(ICQ_ASSERT_CAPS));
+	g_string_append(url, "interestCaps=&");
+	g_string_append_printf(url, "events=%s&", purple_url_encode(ICQ_EVENTS));
+	g_string_append_printf(url, "includePresenceFields=%s", purple_url_encode(ICQ_PRESENCE_FIELDS));
```
Comment (EionRobb) [inline libicyque.c:1637]: This call doesn't need the sig_sha256 hash?

File: libicyque.c (hunk 1 above)
Comment (isra17) [reply libicyque.c:35]: The files seems to have a mix of DOS and Unix line terminator, VIM then pick it up as a DOS type of file and create new lines as such. Would you accept a PR that runs `clang-format` on this file and fix a few issues with the file (Mix of tabs and spaces, newlines, etc)?

File: libicyque.c (hunk 2 above)
Comment (isra17) [reply libicyque.c:1637]: Not anymore, or at least, not on this API endpoint, which has been taken from the Web app.

**Pull request merged**